Website/Providers/DataProviders/SqlDataProvider/07.04.00.SqlDataProvider

```diff
@@ -691,7 +695,7 @@ BEGIN
         'Reject',
         'Reject',
         NULL,
-        1,
+        2,
```

Comment:
PS: and it broke update for users of my Turbo scripts, which established an unique index on these columns